Objects/frameobject.c

```diff
@@ -649,6 +649,7 @@ frame_dealloc(PyFrameObject *f)
 static inline Py_ssize_t
 frame_nslots(PyFrameObject *frame)
 {
+    //return frame->f_code->co_nfastlocals;
```

Comment:
Leftover?
```suggestion
```

Reply:
@markshannon, before b11a951f16 this function really was equivalent to `frame->f_code->co_nfastlocals`, but you changed it to include the frame's stack.  Was this intentional?  Also, are `frame_traverse()` and `frame_tp_clear()` doing extra work then?

Reply:
Mark explained that now we track builtins/globals/locals too, hence the difference.